scripts/zkai_attention_kv_fused_softmax_table_microprofile_gate.py

```
@@ -0,0 +1,877 @@
+
+def write_tsv(path: pathlib.Path, payload: dict[str, Any]) -> None:
+    validate_payload(payload)
+    path.parent.mkdir(parents=True, exist_ok=True)
+    text = to_tsv(payload, validate=False)
+    with tempfile.NamedTemporaryFile("w", encoding="utf-8", newline="", dir=path.parent, prefix=f".{path.name}.", suffix=".tmp", delete=False) as handle:
+        tmp = pathlib.Path(handle.name)
+        handle.write(text)
+    try:
+        loaded_rows = list(csv.DictReader(tmp.read_text(encoding="utf-8").splitlines(), delimiter="\t"))
+        if len(loaded_rows) != len(payload["profile_rows"]):
+            raise FusedSoftmaxTableMicroprofileGateError("TSV row count drift")
```

Ruff (0.15.12)

[warning] Abstract `raise` to an inner function

(TRY301)